lib/src/code/code.dart

```diff
@@ -63,23 +75,26 @@ class Code {
     return result;
   }
 
-  static List<String> _getCommentWords(
-    String str,
-    List<String> singleLineComments,
-  ) {
-    for (final singleLineComment in singleLineComments) {
-      final commentIndex = str.indexOf(singleLineComment);
+  static String? _getCommentSignByLanguage(Mode? language) {
+    String? commentSign;
+    if (slashCommentSignLanguages.contains(language)) {
+      commentSign = '//';
+    } else if (language == python) {
+      commentSign = '#';
+    }
+    return commentSign;
+  }
 
-      if (commentIndex == -1) {
-        continue;
-      }
+  static List<String> _getCommentWords(String str, String? singleLineComment) {
+    if (singleLineComment != null && str.contains(singleLineComment)) {
+      final commentIndex = str.indexOf(singleLineComment);
 
       return str
           .substring(commentIndex + singleLineComment.length)
           .split(RegExp(r'\s+')); // Split by any whitespaces.
```

Comment:
You deleted this comment and then re-created it. So in git history it is now yours and not mine. History is very important when looking for what introduced errors. So when you revert the changes, you should squash the commits and publish a new branch.